.backportrc.json

```
@@ -1,8 +1,20 @@
 {
   "upstream": "elastic/ecs",
-  "branches": [{ "name": "1.0", "checked": true }],
-  "all": true,
-  "multipleCommits": false,
-  "multipleBranches": true,
-  "labels": ["backport"]
-}
\ No newline at end of file
+  "branches": [
+    { "name": "master", "checked": true },
+    { "name": "1.x", "checked": true },
+    "1.6",
+    "1.5",
+    "1.4",
+    "1.3",
+    "1.2",
+    "1.1",
+    "1.0"
+  ],
+  "targetPRLabels": ["backport"],
```

We're pretty reliable on tagging with `needs_backport` when needed.  Perhaps we could add the following as well?

```suggestion
  "targetPRLabels": ["backport"],
  "prFilter": "needs_backport",
```

---

I like the idea. I adjusted the syntax a bit; `prFilter` let's you define an entire [GitHub query](https://docs.github.com/en/github/searching-for-information-on-github/understanding-the-search-syntax) if we want to get more elaborate in our filtering in the future.